crates/orchestrator/src/tests/jobs/mod.rs

```diff
@@ -98,10 +98,6 @@ async fn create_job_job_exists_in_db_works() {
     let database_client = services.config.database();
     database_client.create_job(job_item.clone()).await.unwrap();
 
-    assert!(
-        create_job(JobType::ProofCreation, "0".to_string(), HashMap::new(), services.config.clone()).await.is_err()
-    );
```

Comment:
check it returns ok instead and check DB that job isn't there imo

Reply:
So basically this should should fail because previously we were throwing an error. but now we just log a warning and safe exit from the create_job function.

Reply:
In my opinion the is_ok check is not needed here.

Reply:
we can remove is ok check but i think checking it's not present in the DB makes sense, especially because now the code isn't throwing an error

Reply:
in this test the job is being created above in the first line. After that I can call this function and check in the db that there is only one job. wdyt about this ?

Reply:
ya ig that works but why not just do `get_job_by_internal_id_and_type`? you already have the db client and the fn is already there